hudi-sync/hudi-hive-sync/src/main/java/org/apache/hudi/hive/HiveSyncConfig.java

```diff
@@ -69,6 +70,7 @@ public static String getBucketSpec(String bucketCols, int bucketNum) {
 
   public HiveSyncConfig(Properties props) {
     super(props);
+    validateParameters();
```

Comment:
since validation is done in constructor. we don't need to check in JDBCExecutor either right?

Reply:
agree, deleted check in JDBCExecutor and QueryBasedDDLExecutor.